### PR TITLE
Docs re `tail -f` and `--records-per-batch 1`

### PR DESCRIPTION
**Pull request by johnkerl**
For #1178

### CODE REVIEW

File: docs/src/manpage.md [hunk 1]
```diff
@@ -1858,9 +1860,10 @@ MILLER(1)                                                            MILLER(1)
 
        -i             Use interpolated percentiles, like R's type=7; default like type=1.
                       Not sensical for string-valued fields.\n");
-       -s             Print iterative stats. Useful in tail -f contexts (in which
+       -s             Print iterative stats. Useful in tail -f contexts, in which
```
Comment (zmajeed) [inline docs/src/manpage.md:1863]: code

File: docs/src/manpage.md [hunk 2]
```diff
@@ -1896,9 +1899,10 @@ MILLER(1)                                                            MILLER(1)
                       There must be an even number of names.
        -g {e,f,g}     Optional group-by-field names.
        -v             Print additional output for linreg-pca.
-       -s             Print iterative stats. Useful in tail -f contexts (in which
+       -s             Print iterative stats. Useful in tail -f contexts, in which
```
Comment (zmajeed) [inline docs/src/manpage.md:1902]: code

File: docs/src/manpage.txt [hunk 3]
```diff
@@ -1837,9 +1839,10 @@ MILLER(1)                                                            MILLER(1)
 
        -i             Use interpolated percentiles, like R's type=7; default like type=1.
                       Not sensical for string-valued fields.\n");
-       -s             Print iterative stats. Useful in tail -f contexts (in which
+       -s             Print iterative stats. Useful in tail -f contexts, in which
```
Comment (zmajeed) [inline docs/src/manpage.txt:1842]: code

File: docs/src/manpage.txt [hunk 4]
```diff
@@ -1875,9 +1878,10 @@ MILLER(1)                                                            MILLER(1)
                       There must be an even number of names.
        -g {e,f,g}     Optional group-by-field names.
        -v             Print additional output for linreg-pca.
-       -s             Print iterative stats. Useful in tail -f contexts (in which
+       -s             Print iterative stats. Useful in tail -f contexts, in which
```
Comment (zmajeed) [inline docs/src/manpage.txt:1881]: code

File: docs/src/reference-verbs.md [hunk 5]
```diff
@@ -3235,9 +3238,10 @@ accumulated across the input record stream.
                There must be an even number of names.
 -g {e,f,g}     Optional group-by-field names.
 -v             Print additional output for linreg-pca.
--s             Print iterative stats. Useful in tail -f contexts (in which
+-s             Print iterative stats. Useful in tail -f contexts, in which
```
Comment (zmajeed) [inline docs/src/reference-verbs.md:3241]: code

File: internal/pkg/transformers/stats1.go [hunk 6]
```diff
@@ -55,11 +55,12 @@ Options:
 
 -i             Use interpolated percentiles, like R's type=7; default like type=1.
                Not sensical for string-valued fields.\n");
--s             Print iterative stats. Useful in tail -f contexts (in which
+-s             Print iterative stats. Useful in tail -f contexts, in which
```
Comment (zmajeed) [inline internal/pkg/transformers/stats1.go:58]: code

**Comment by zmajeed** [comment]
looks great - a note or link to the streaming doc in the  `--records-per-batch` entry in reference-main-flag-list.md could be helpful

**Comment by johnkerl** [comment]
@zmajeed good call, thank you! :)

https://github.com/johnkerl/miller/commit/f528e65802158bdd91e9ee76a33838465fb9938a

**Pull request merged**